lib/maril/generator.rb

```diff
@@ -19,7 +19,7 @@ def generate
       end
       cmd << app['container']['docker']['image']
       cmd << app['cmd'] if app['cmd']
-      cmd.join(" \\\n")
+      cmd.join(" ")
```

Comment:
I see your point. I guess it's because of the escape problem. I will try if I can properly escape the problem character.